VTEX - Logistics API.json

```diff
@@ -4186,7 +4220,118 @@
+                                                    "type": "string",
+                                                    "description": "Delivery channel type.",
+                                                    "enum": [
+                                                        "delivery",
+                                                        "pickup"
+                                                    ]
+                                                }
+                                            },
+                                            "timeToRefill": {
+                                                "type": "string",
+                                                "description": "Time to refill in the format `HH:MM:SS`.",
+                                                "nullable": true
+                                            },
+                                            "dateOfSupplyUtc": {
+                                                "type": "string",
+                                                "description": "Date and time scheduled for the SKU from the warehouse to have quantity and availability for sales again. The format is [ISO 8601 time zone offset](https://learn.microsoft.com/en-us/rest/api/storageservices/formatting-datetime-values), as in `YYYY-MM-DDThh:mm:ss.ssZ`."
+                                            },
+                                            "supplyLotId": {
+                                                "type": "string",
+                                                "description": "Supply lot ID."
+                                            },
+                                            "keepSellingAfterExpiration": {
+                                                "type": "boolean",
+                                                "description": "Defines if the SKU from the warehouse should be available for sales after running _out-of-stock_ (`true`), or not (`false`)."
+                                            },
+                                            "transfer": {
+                                                "type": "string",
+                                                "description": "Transfer system.",
+                                                "nullable": true
+                                            },
+                                            "leadTime": {
+                                                "type": "string",
+                                                "description": "[Lead time](https://help.vtex.com/en/tutorial/lead-time-shipping-time-at-sku-level--16yv5Mkj6bTyWR1hCN2f4B) is an optional time configuration for a SKU in a warehouse. It can be handling time, fabrication or how long it takes for the item to be available to be shipped to customers. The lead time is a part of the total shipping time and is considered for shipping date calculation. The format is `DD.HH:MM:SS`.",
```

Comment:
```suggestion
                                                "description": "[Lead time](https://help.vtex.com/en/tutorial/lead-time-shipping-time-at-sku-level--16yv5Mkj6bTyWR1hCN2f4B) is an optional time configuration for an SKU in a warehouse. It can be handling time, fabrication or how long it takes for the item to be available to be shipped to customers. The lead time is a part of the total shipping time and is considered for shipping date calculation. The format is `DD.HH:MM:SS`.",
```

Reply:
The Localization team recommends using "a SKU", as discussed in this thread: https://vtex.slack.com/archives/C052CEU8GDU/p1717767198746449